package.json

```diff
@@ -3,9 +3,14 @@
   "private": true,
   "description": "A collection of components that make it easy to build interactive charts with D3",
   "scripts": {
-    "postinstall": "lerna bootstrap",
-    "publish": "lerna publish --conventional-commits --message \"chore: publish\" --yes",
-    "test": "lerna run test --stream --concurrency 1"
+    "postinstall": "./node_modules/.bin/lerna bootstrap",
```

Comment:
This is me being a bit old fashioned, to avoid people having to -g install lerna I included the local path, but I can change this to npx now that it default ships with node ( and has done for a few years 😅 )